internal/app/go-filecoin/paymentchannel/types.go

```diff
@@ -1,18 +1,43 @@
 package paymentchannel
 
 import (
+	"reflect"
+
 	"github.com/filecoin-project/go-address"
 	"github.com/filecoin-project/specs-actors/actors/builtin/paych"
 )
 
+//go:generate cbor-gen-for ChannelInfo VoucherInfo
+
+// The key for the store is the payment channel's "robust" or "unique" address
+
 // ChannelInfo is the primary payment channel record
+// IDAddr: internal ID address for payment channel actor
```

Comment:
This should be unnecessary since you can use `state.View.InitResolveAddress` to retrieve IDAddr from UniqueAddr.